source/_components/mqtt.markdown

```diff
@@ -119,6 +119,10 @@ mqtt:
 There is an issue with the Mosquitto package included in Ubuntu 14.04 LTS. Specify `protocol: 3.1` in your MQTT configuration to work around this issue.
 </p>
 
+<p class='note'>
+If you are running a mosquitto instance on a different server with proper SSL encryption using a service like letsencrypt you may have to set the certificate to the operating systems own `.crt` certificates file. In the instance of ubuntu this would be `certificate: /etc/ssl/certs/ca-certificates.crt`
```

Comment:
In MQTT, `certificate` should point to a file containing the root of the certificate chain that signed the certificate you are using for SSL with Mosquitto.  Mine points at `identrust-root-and-lets-encrypt-x3.pem`, but if that same certificate is in Ubuntu's system store, pointing `certificate` at it will work and is probably better.

I contributed #962 that clarified what `certificate` is used for after I traced the code to see what exactly it does. The initialization code is https://github.com/home-assistant/home-assistant/blob/4692ea85b7e53ab21cefc9bfe4b73fc436d5238d/homeassistant/components/mqtt/__init__.py#L334 and the underlying documentation is https://pypi.python.org/pypi/paho-mqtt/1.1 (search for tls_set)

```
ca_certs
    a string path to the Certificate Authority certificate files that are to be treated as trusted by this client. If this is the only option given then the client will operate in a similar manner to a web browser. That is to say it will require the broker to have a certificate signed by the Certificate Authorities in ca_certs
```

I'm assuming Jammy has tested this and that setting `certificate` this way made it work better for him.  If that's true, I approve.